src/main/java/com/suse/salt/netapi/calls/modules/Match.java

```diff
@@ -47,6 +76,39 @@
                 Optional.of(args), new TypeToken<Boolean>(){});
     }
 
+    /**
+     * Return True if the minion matches the given pillar target.
+     * The delimiter argument can be used to specify a different delimiter.
+     * @param tgt
+     * @param delimiter
```

Comment:
As you added javadoc comments I might suggest you to also add for completeness short descriptions of all the parameters as it is usually done, for example:

```java
@param tgt pillar target
@param delimiter optional different delimiter
```

Reply:
I will add it.

Reply:
@renner, I just pushed everything... I hope it can me merged :)